src/Entity/Package.php

```diff
@@ -33,6 +33,13 @@
 use Composer\Util\HttpDownloader;
 use DateTimeInterface;
 
+enum PackageFreezeReason: string
```

Comment:
why defining it inline instead of making it autoloadable ?

Reply:
Because it belongs together with the class, and is much more readable to me than having tiny enum files lingering around, and there is no reason to use it without package instance, so the package class will be loaded when you need it.